test/algorithms/buffer/buffer_multi_polygon.cpp

```diff
@@ -637,10 +634,7 @@ void test_all()
 
     TEST_BUFFER(rt_w20, join_miter, end_flat, 63.0269, 1.0);
     TEST_BUFFER(rt_w21, join_miter, end_flat, 26.3137, 1.0);
-
-#if defined(BOOST_GEOMETRY_TEST_FAILURES)
-    TEST_BUFFER(rt_w22, join_miter, end_flat, 86.0416, 1.0);
-#endif    
+    TEST_BUFFER(rt_w22, join_miter, end_flat, 86.1274, 1.0);
```

Comment:
There is a change in area here. Did the expected result changed?

Reply:
👍  Yes, the earlier area was wrong.

This is the new result:

![image](https://github.com/user-attachments/assets/33342bf7-3c84-41ac-b961-c9c52d58cc23)


While earlier (however, not in `develop` but probably in one of my branches) it did not include:

![image](https://github.com/user-attachments/assets/87309708-34b3-40b4-9885-58ca916a3870)